### DISCUSSION TITLE
Example for zero trust with oathkeeper?

### DISCUSSION THREAD

**Discussion by JasonCubic** [ory/hydra#2305] [Q&A] [answered]
Is there an example anywhere showing how to connect oathkeeper to hydra?  maybe using the hydra-login-consent-node project from the docs?  I'm new to all of this and having a hard time figuring out how to connect them.  I'm guessing we need a hello world kind of api to protect as well.

**Comment by vinckr** [comment]
Hey Jason, 
we don't have a definite guide to connect Ory Hydra and Ory Oathkeeper, there is some documentation: 
https://www.ory.sh/oathkeeper/docs/oauth2-token-introspection-hydra. 
All the other pieces should be there in the docs. 
If you want to collaborate on a `hello word` style example, I would be happy to support you.

**Comment by JasonCubic** [comment, marked as answer]
Okay great.  I'll start working on this and see how far I can get before getting stuck.

**Reply by vinckr** [reply]
Sounds good! 
Feel free to contact me in [slack](ory.sh/chat) or via email (vincent[ät]ory[dot]sh) if you need assistance.

**Comment by hiendaovinh** [comment]
Is there any updates regarding this? Thank you!

**Reply by vinckr** [reply]
Hello @hiendaovinh 
check out our community Oathkeeper examples https://github.com/ory/examples/tree/master/oathkeeper

**Comment by raolak** [comment]
Hello, Any reference/pointers for  ory oAuthkeeper, hydra and Kratos working examples?  I know that hydra now integrates with Kratos, but there is no examples or documents. Appreciate if one can share the example/reference docs. Thank You !

**Reply by vinckr** [reply]
Hello @raolak 
check out this discussion:
https://github.com/ory/kratos/discussions/2976
You can find more examples for Oathkeeper and self-hosting Ory services here: https://github.com/ory/examples
